### ISSUE TITLE
minimum horizontal resolution increased after update beyond 1280 pixels

### ISSUE THREAD

**Issue by Newk77** [kmatheussen/radium#1349]
### 2. Which version of Radium are you using? Do you know if an earlier version has worked?

Before the update i was on 6.9.94 where i could close either the instrument view or editor view in order to make it fit on screen.
Now after update to 6.9.96 this minimum horizontal resolution is higher then 1280 pixels.

Why is this a thing if there are sliders anyway, or am i missing something (new) in the settings?

### 3. If relevant, which operating system did you run Radium on?
Running linux version on LXDE (lightweight desktop environment)

**Comment by kmatheussen** [comment]
> edit: Seems the bottom bar is the culprit? I somehow set the LinesPerBeat dialoge down there to hidden and now it fits.. but i rather hide some other less used things, how do i get it back and hide others? (i rather hide the octave switcher as i use a midi keyboard)

Yeah, it should be configurable what you want to see in the bottom bar. I gather you don't want to use a smaller font?

Anyway, until then there is actually a way to hide elements from the bottom bar:

1. Select "New Scheme listener" from the windows menu.
2. Paste  `(<gui> :hide (<gui> :child -1 "octave_label"))` to remove the octave label.

Name of other elements to remove:
`octave_up_button`
`octave_down_button`
`drunk_velocity_onoff`
and so forth.
You can find the name of all elements by opening this file: `https://raw.githubusercontent.com/kmatheussen/radium/master/Qt/qt4_bottom_bar_widget.ui` in the Qt designer.

**Comment by kmatheussen** [comment]
Here's some more names of elements:

`LPBWidget`
`BPMWidet`
'SignatureWidget'

**Comment by kmatheussen** [comment]
You can also edit your `~/.radium/keybindings.conf` file. For instance by adding lines like this:

```
*
ra.evalScheme('(<gui> :hide (<gui> :child -1 "octave_label"))')
ra.evalScheme('(<gui> :hide (<gui> :child -1 "LPBWidget"))')
*
```

**Comment by Newk77** [comment]
"New Scheme listener" does not stay open... closes immidiatly
error in terminal: `xterm: cannot load font "-misc-fixed-medium-r-semicondensed--13-120-75-75-c-60-iso10646-1"`
[this](https://forum.manjaro.org/t/xterm-missing-default-fonts/35113) probably is the reason, re-installing the mentioned package did not fix it for me.

HOWEVER; making a `~/.radium/keybindings.conf` DID WORK :D

```
*
ra.evalScheme('(<gui> :hide (<gui> :child -1 "octave_up_button"))')
ra.evalScheme('(<gui> :hide (<gui> :child -1 "octave_down_button"))')
ra.evalScheme('(<gui> :hide (<gui> :child -1 "drunk_velocity_onoff"))')
*
```
keeping the octave label is still handy as switching with F1 and F2 still works :P
i tried to get rid of the tiny sequencer timeline on the bottom bar and show the ones for BPM features with the help of the source in your link but could not find it, also `ra.evalScheme('(<gui> :show (<gui> :child -1 "LPBWidget"))')` probably isn't correct syntax :P 

cheers for the help!

**Comment by Newk77** [comment]
ah! never mind about setting BPM widget to 'show'.... just now noticed it toggles automaticly to show when deselecting it from the timing-track of the editor, doh! (need to learn more about the trackers functions i guess)

But would really would like to know how to disable the mini sequencer overview from the bottom bar... i rather use the sequencer window-part itself for that.
